onset_maproom/docker/httpd.conf

```diff
@@ -0,0 +1,25 @@
+LoadModule mpm_prefork_module modules/mod_mpm_prefork.so
+LoadModule unixd_module modules/mod_unixd.so
+LoadModule authz_core_module modules/mod_authz_core.so
+LoadModule log_config_module modules/mod_log_config.so
+
+User apache
+Group apache
+Listen 8050
```

Comment:
Whoops, missed this until I read the dlconfig PR. Please use port 8000 like all the other apps do.

Reply:
Has anything else in this file been edited from the BDFloodsMR version?

Reply:
nothing besides the name fbfmaproom which doesn't really appear to matter